### PR TITLE
Subfamilies

### PR DESCRIPTION
**Pull request by gustavo-salazar**
Adds a modifier `subfamilies`. It allows listing the panther subfamilies.

* Dependencies updates and test updates to work on most recent versions of selenium.

### CODE REVIEW

**Comment by gustavo-salazar** [comment]
@matthiasblum I tested this with the new test DB and after a couple of fixes I think is ready for revision. SO please check whenever you have some time

File: webfront/views/modifiers.py [hunk 1]
```diff
@@ -853,5 +853,15 @@ def get_model_structure(value, general_handler):
     return get_model_structure
 
 
+def get_subfamilies(value, general_handler):
+    queryset = general_handler.queryset_manager.get_queryset().first()
+    entries = Entry.objects.filter(integrated=queryset.accession, source_database='panther', is_alive=False)
```
Comment (matthiasblum) [inline webfront/views/modifiers.py:858]: We should not filter by `source_database` as this modifier will be used for FunFam.

**Review by matthiasblum — CHANGES_REQUESTED** [review]
Looks pretty good. There only one small thing to change so the modifier can be used for CATH-Gene3D superfamilies - FunFams.

File: webfront/views/modifiers.py (hunk 1 above)
Comment (gustavo-salazar) [reply webfront/views/modifiers.py]: Change made. I also added a fixture for funfams and updated the tests, please check 1fdbf66

**Review by matthiasblum — APPROVED** [review]
All good, thank you.

**Pull request merged**